styles.css

```diff
@@ -0,0 +1,520 @@
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 16px;
+}
+
+body {
+	font-family: 'Gelasio', serif;
+	color: #0E1116;
+	background-color: #374B68;
+}
+
+header {
+	background-color: #374B68;
+	padding-top: 16px;
+	height: 160px;
+	width: 100%;
+	position: fixed;
+	top: 0;
+	z-index: 1;
+}
+
+.header-top {
+	text-align: center;
+	overflow: hidden;
+}
+
+.logo {
+	display: inline-block;
+	height: 80px;
+	margin-right: 30px;
+}
+
+#burger-menu {
+	display: none;
+}
+
+h1 {
+	display: inline-block;
+	padding: 0;
+	margin: 0;
+	font-size: 65px;
+	font-weight: 400;
+}
+
+h2 {
+	font-size: 40px;
+	margin: 0;
+	padding: 20px 0;
+	height: 80px;
+	text-transform: uppercase;
+	font-weight: 400;
+	color: #C0D7E0;
+}
+
+h3 {
+	display: block;
+	padding: 0;
+	margin: 0;
+	font-size: 30px;
+	font-weight: 400;
+}
+
+h4 {
+	margin: 0;
+	padding: 0;
+	font-size: 28px;
+}
+
+.header-top h1 {
+	position: relative;
+	bottom: 17px;
+	color: #20385E;
+}
+
+.accent-color {
+	color: #C0D7E0;
+}
+
+.accent-largetext {
+	font-size: 38px;
+	font-weight: 500;
+}
+
+.accent-serif {
+	font-family: 'Gelasio', serif;
+}
+
+nav {
+	background-color: #C0D7E0;
+	padding: 10px 0;
+	overflow: auto;
+}
+
+a {
+	display: inline-block;
+	text-decoration: none;
+	color: #0E1116;
+}
+
+.nav-links {
+	overflow: auto;
+	width: 800px;
+	margin: 0 auto;
+	padding: 0 30px;
+	font-size: 20px;
+	font-family: 'Poppins', sans-serif;
+	text-transform: uppercase;
+}
+
+nav ul {
+	list-style-type: none;
+	margin: 0;
+	padding: 0;
+	overflow: hidden;
+	line-height: 23px;
+}
+
+nav li {
+	display: block;
+	float: left;
+	width: 33.2%;
+	text-align: center;
+	margin: 2px 0 0 0;
+}
+
+section.showcase, section.about, section.portfolio {
+	height: calc(100vh - 235px);
+	position: relative;
+}
+
+section.showcase {
+	background-image: url('./images/project-1-hero.jpg');
+    background-position: center;
+    background-repeat: no-repeat;
+	background-size: cover;
+	text-align: center;
+	margin-top: 160px;
+}
+
+.hero-caption {
+	display: inline-block;
+	text-align: center;
+	position: absolute;
+	width: 575px;
+	top: 45%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	color: #E6FAFC;
+	background-color: rgba(14, 17, 22, 0.7);
+	padding: 20px 50px;
+}
+
+.hero-caption h1 {
+	font-size: 95px;
+}
+
+.a-box {
+	display: inline-block;
+	text-align: center;
+	color: #E6FAFC;
+	background-color: rgba(14, 17, 22, 0.7);
+	padding: 15px 30px;
+}
+
+.a-section-title {
+	margin-left: 20px;
+}
+
+.a-section-anchor {
+	position: absolute;
+	margin-top: -160px;
+	outline: none;
+}
+
+section.showcase .a-box {
+	font-size: 20px;
+	position: absolute;
+	bottom: 10%;
+	left: 70%;
+	transform: translate(-50%, 0);
+}
+
+section.about {
+	background-color: #374B68;
+	min-height: 600px;
+}
+
+.center-container {
+	width: 880px;
+	margin: 0 auto;
+	height: 100%;
```

Comment:
check this is doing something, if not remove it

Reply:
Yes - it stretches out the portfolio section content to fill the whole section height